runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -159,6 +159,13 @@
 
   void setEnableMetrics(Boolean enableMetrics);
 
+  @Description(
+      "Use Flink accumulators to store the metrics. If turned off, metrics will still be reported but can't be queried via PipelineResult. This saves network and memory.")
+  @Default.Boolean(true)
+  Boolean getUseMetricAccumulator();
```

Comment:
With the default problem why not `getDisableMetricAccumulator()` with default `false`?

Reply:
Generally the default for boolean is false and there should be no need to specify it here.

Reply:
Copy & paste oversight.

Reply:
Note that the default for `boolean` is `false` and for `Boolean` it is null. I want to follow the convention here to return an object, as all other getters/setters do. So I'm re-adding the default value of false which is also more explicit for users who are not experienced Java programmers.